molfeat/calc/fingerprints.py

```diff
@@ -139,58 +147,53 @@
     # COUNTING FP
     "ecfp-count": {
         "radius": 2,  # ECFP4
-        "nBits": 2048,
-        "invariants": [],
-        "fromAtoms": [],
-        "useChirality": False,
+        "fpSize": 2048,
+        "includeChirality": False,
         "useBondTypes": True,
-        "useFeatures": False,
-        "includeRedundantEnvironments": False,
+        "countSimulation": False,
```

Comment:
is this a match for match for the datamol version ? Otherwise, might be worth it to compared fingerprints before and after to changes to make sure we are reproducing the old version so there is no impact for end users.

Reply:
Reverted to the original configuration.